docs/docs/tutorial/part-1/index.mdx

```diff
@@ -301,7 +301,7 @@ To connect your code on GitHub to your Gatsby Cloud account, do the following:
 
    ![The GitHub permissions page, asking whether you want to give Gatsby Cloud access to your repos. The "All repositories" option is selected.](./03-github-gatsby-cloud-permissions.png)
 
-1. Now, when you go back to the Gatsby Cloud window, the repository list should contain your GitHub repository. Select it.
+1. Now, when you go back to the Gatsby Cloud window, the repository list should contain your GitHub repository. Select it or click on **"Import"**.
```

Comment:
```suggestion
1. Now, when you go back to the Gatsby Cloud window, the repository list should contain your GitHub repository. Select it by clicking on **"Import"**.
```

Since it actually says "Import", let's point people to that :)